chain/indexer/src/handlers.rs

```diff
@@ -2162,7 +2162,7 @@ pub mod tests {
                 OutgoingChannelClosureInitiatedFilter::signature(),
                 H256::from_slice(&channel.get_id().to_bytes()),
             ],
-            data: Vec::from(U256::from(timestamp.duration_since(UNIX_EPOCH).unwrap().as_secs()).to_bytes()).into(),
+            data: Vec::from(U256::from(timestamp.saturating_sub(UNIX_EPOCH).as_secs()).to_bytes()).into(),
```

Comment:
This can never fail though, since `UNIX_EPOCH` is 0.

Reply:
I agree. But as behaviour would be the same, I prefer to use most of the time, where possible.